App/UI/Validator.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace App.UI
+{
+    public class Validator
+    {
+        private static int ParseIntFromInput(string input)
+        {
+            return Int32.TryParse(input, out int num) ? num : -1;
+        }
+        
+        public static bool IsGameModeValid(string input)
+        {
+            int option = ParseIntFromInput(input);
+            // return Enumerable.Range(1, 3).Contains(option);
+            return Enumerable.Range(1, 1).Contains(option);
+        }
+
+        public static bool IsBoardSizeValid(string input)
+        {
+            int option = ParseIntFromInput(input);
+            return Enumerable.Range(3, 3).Contains(option);
```

Comment:
it's a bit weird to use a magic number here when we can use the board Enumerable to check if the options are valid. if we wanted to add more board sizes you would need to change the code in two different places.

since the current work is to focus on 3x3, I think you could 1. update the enum to only have 3x3 as an option and then update this code to use that as a validation instead of `Enumerable.Range(3, 3)`